generator/test_case_generator.go

```diff
@@ -122,6 +122,10 @@ var testTypeToTestConfig = map[string][]testConfig{
 		{testDir: "../../test/performance/statsd"},
 		{testDir: "../../test/performance/collectd"},
 	},
+	"ec2_windows_performance": {
```

Comment:
Nice! I like that this is specific to the platform similar to other integration tests.